runner.go

```diff
@@ -538,3 +540,22 @@ func expandStringSlice(slice []string) []string {
 	}
 	return result
 }
+
+// writeRunID writes the run ID to a file if the flag is set
+func writeRunID(c cliContext, runStatus *rainforest.RunStatus) error {
+	filePath := c.String("save-run-id")
+	if filePath == "" {
+		return nil
+	}
+
+	file, err := os.Create(filePath)
```

Comment:
Does `os.Create` work with `filePath` having the value `/dev/stderr` / `/dev/stdout`? Not a big deal if it doesn't, but it'd be nice if the CLI accepts these special files.

Reply:
Weirdly enough, it works... _once_. And then it completely breaks the `--save-run-id` flag and I need to recompile.

Ah, it turns out that happens regardless of what I pass in, not just for special values like `/dev/stdout` 🤔

Reply:
Just running `./build/rainforest run --help` twice results in `--save-run-id` disappearing for the second execution 🤔